package.json

```diff
@@ -29,7 +30,8 @@
     "react-dom": "^17.0.1",
     "react-feather": "^2.0.8",
     "react-p5": "^1.3.6",
-    "theme-ui": "^0.3.1",
+    "sharp": "^0.28.2",
```

Comment:
What are we using this for? I don't see any usage of sharp in the code and nothing else in yarn.lock depends on it.

If you do need it, heads up that it's currently not working in ARM Macs, so let's use an alternative if we can.

Reply:
**Hey, it does work in arm macs, I have tested it on mine :)**

So when I ran the project, it required me to install sharp, it was saying that can't find sharp, so have to import it to make it work.

I think some package is using it as a dependency, but still, it does work on arm Macs.

Reply:
That's weird

Reply:
found out, next js has an optional dependency on it, but sometimes not having it causes it to break ( what happened in my case )

Reply:
interesting, probably related to the image optimization

Reply:
yeah :)

Reply:
It works on my m1 by failing to install sharp & continuing without it as a dependency:

<img width="1376" alt="Screen Shot 2021-05-20 at 15 50 19" src="https://user-images.githubusercontent.com/5891442/119040263-2f477a80-b983-11eb-8b60-0d4ef59f759e.png">

Reply:
So, when I remove sharp this is what I get:
![image](https://user-images.githubusercontent.com/41180869/119093732-b3563c80-ba2d-11eb-893d-049439773305.png)

And this is how my page looks
![image](https://user-images.githubusercontent.com/41180869/119093792-c5d07600-ba2d-11eb-9420-728b840e556e.png)

Now very strangely, when I install sharp, although I just saw that its build failed ( **sorry for saying earlier that it build successfully, I just noticed it** ), but installing it **_somehow fixes everything for me_**

Reply:
Hey @maxwofford, just saw your PR and It has fixed this issue, I have merged it and now we longer have sharp as a dependency 🚀 

Thanks you are awesome 🙌